### PR TITLE
Feature/k8s

### PR DESCRIPTION
**Pull request by ejsmith**
`helm install ./exceptionless --name exceptionless-dev --namespace dev --values ./exceptionless/values.override.yaml`

### Questions ###
- [x] separate collector and api services?

### Required ###
- [x] get our test cluster out of failed state
- [x] figure out https (nginx ingress / cert-manager)
- [x] configure the rest of the settings
- [x] manually set scale on pods
- [x] set dns entries
- [x] statsd service (remove it from kibana box)
- [x] switch production application to use k8s
- [x] set resource constraints on all pods
- [x] update the appveyor build to automatically deploy the new build with master / non-master separated

### CODE REVIEW

File: src/Exceptionless.Web/Exceptionless.Web.csproj [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Exceptionless.AspNetCore.Signed" Version="4.3.2004" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.4.0" />
     <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.2" />
+    <PackageReference Include="Joonasw.AspNetCore.SecurityHeaders" Version="2.7.0" />
```
Comment (nj) [inline src/Exceptionless.Web/Exceptionless.Web.csproj]: Seems to already exists below... Possible occurred through a merge-conflict or alike

Comment (ejsmith) [reply src/Exceptionless.Web/Exceptionless.Web.csproj]: You are right. Thanks.

**Comment by ejsmith** [comment]
https://docs.microsoft.com/en-us/azure/aks/ingress-tls

**Comment by tekmaven** [comment]
This may be useful for autoscaling: https://github.com/Azure/azure-k8s-metrics-adapter

**Comment by ejsmith** [comment]
Thanks!

**Pull request merged**